mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -31,7 +31,7 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_7910 finishUsing (Lnet/minecraft/class_1937;Lnet/minecraft/class_1309;)Lnet/minecraft/class_1799;
 		ARG 1 world
 		ARG 2 user
-	METHOD method_7911 getOrCreateSubTag (Ljava/lang/String;)Lnet/minecraft/class_2487;
+	METHOD method_7911 getOrCreateSubNbt (Ljava/lang/String;)Lnet/minecraft/class_2487;
```

Comment:
Not sure about this rename. You are working with a sub tag, not some sub NBT anything.

Reply:
hmm NBT should still be somewhere in the name though imo. getOrCreateNbtSubTag?

Reply:
This looks good to me.

Reply:
I'd rather avoid redundancy. If you think `nbt` is necessary, the name you gave is fine.

Reply:
how about getOrCreateNbtSubTag?

Reply:
That works, I guess.

Reply:
Oops lol I didn't notice I came up with that yesterday too and already asked lol.